README.md

```diff
@@ -16,3 +16,93 @@ https://github.com/dwyl/intellectual-property and raise an
 on GitHub if you require clarification!
 
 See: https://github.com/dwyl/contributing/issues/1 + https://github.com/dwyl/summer-2015/issues/27
+
+## Recommended Guidelines
+
+### Labels
```

Comment:
This is a great section but I think that we want the first thing that people see in the contributing guidelines to be OPEN AN ISSUE, because people occasionally forget to do that and it's the basis for the whole thing.

I would move labels to the bottom as a reference item, mostly because people who aren't members of the dwyl org already or of a team with edit rights can't actually add labels to issues anyway 👍